File: src/test/java/com/und/common/utils/DateUtilsTest.java
package com.und.common.utils;

import org.junit.Test;

import java.util.Date;

import static org.assertj.core.api.Assertions.*;

/**
 * Created by shiv on 21/07/17.
 */
public class DateUtilsTest {
    @Test
    public void now() throws Exception {
        assertThat(new DateUtils().now()).isCloseTo(new Date(), 1000);
    }

}

File: src/test/kotlin/com/und/common/utils/DateUtilsTest.kt
package com.und.common.utils

import org.junit.Test
import java.util.*
import org.assertj.core.api.Assertions.*

/**
 * Created by shiv on 21/07/17.
 */
class DateUtilsTest {


    @Test
    fun now() {
        assertThat(DateUtils().now()).isCloseTo(Date(), 1000)
    }

}